File: php/src/AssessService/LengthScorer.php
<?php

declare(strict_types=1);

namespace FactChecker\AssessService;

class LengthScorer implements Scorer
{
    public function calculateScore(string $sentence): int
    {
        $length = $this->findLength($sentence);

        if ($length > 100) {
            return 2;
        }

        if ($length > 1) {
            return 1;
        }

        return 0;
    }

    private function findLength(string $sentence): int
    {
        return mb_strlen($sentence);
    }
}

File: java/app/src/main/java/factchecker/AssessService/LengthScorer.java
package factchecker.AssessService;

public class LengthScorer implements Scorer {
    @Override
    public Integer calculateScore(String sentence) {
        int length = findLength(sentence);

        if (length > 100) {
            return 2;
        }

        if (length > 1) {
            return 1;
        }

        return 0;
    }

    private Integer findLength(String sentence) {
        return sentence.length();
    }
}
